Update web/css/align-items values explanations (#29686)
mdn/content: files/en-us/web/css/align-items/index.md

    - The property doesn't apply to block-level boxes, and to table cells.

- `flex-start`

>   - : Only used in flex layout. Aligns the flex items to be flush with the edge of the alignment container corresponding to the flex container's main-start or cross-start side, as appropriate.

Reviewer: ```suggestion
  - : Used in flex layout only, aligns flex items to be flush with the flex container's cross-start edge of the flex-line.
```

- `flex-end`

>   - : Only used in flex layout. Aligns the flex items to be flush with the edge of the alignment container corresponding to the flex container's main-end or cross-end side, as appropriate.

Reviewer: ```suggestion
  - : Used in flex layout only, aligns the flex items flush against the flex container's main-end or cross-end side.
```

- `baseline`, `first baseline`, `last baseline`
  - : All flex items are aligned such that their [flex container baselines](https://drafts.csswg.org/css-flexbox-1/#flex-baselines) align. The item with the largest distance between its cross-start margin edge and its baseline is flushed with the cross-start edge of the line.
- `stretch`

>   - : If the items are smaller than the alignment container, auto-sized items will be equally enlarged to fill the container, respecting the width and height limits.

Reviewer: ```suggestion
  - : If the items are smaller than the alignment container, auto-sized items will be equally enlarged to fill the container, respecting the items' width and height limits.
```

>   - : If the combined size of the items is less than the size of the alignment container, any auto-sized items have their size increased equally (not proportionally), while still respecting the constraints imposed by `max-height`/`max-width` (or equivalent functionality), so that the combined size exactly fills the alignment container.

Reviewer: same issue
